docs/UserGuide.md

```diff
@@ -263,8 +291,8 @@ Eg. `t/friend t/family`
 <box type="definition" icon=":fa-solid-book:" light>
 
 <md>**Examples:**</md>
-* `add n/John Doe e/johnd@example.com telegram/@john github/swag-john33` adds John Doe's contact details with no tags.
-* `add n/Betsy Crowe t/friend e/betsycrowe@example.com telegram/@Betsy t/criminal github/betsy-29` adds Betsy Crowe's contact details with a friend and criminal tag.
+* `add name/John Doe email/johnd@example.com telegram/@john github/swag-john33` adds John Doe's contact details with no tags.
+* `add n/Betsy Crowe t/friend e/betsycrowe@example.com tele/@Betsy t/criminal g/betsy-29` uses shortcut commands to add Betsy Crowe's contact details with a friend and criminal tag.
```

Comment:
I feel it would be better to say "shortcut parameters" than "shortcut commands" because in this case, only the short form of the parameters are used and not the command itself.

Reply:
right good catch